src/Swan.Samples/Program.cs

```diff
@@ -154,6 +170,8 @@ private static async Task TestTerminalOutputs()
             }
 
             if (Terminal.ReadKey("Press a key to output the current codepage. (X) will exit.").Key == ConsoleKey.X) return;
+            // Although .NET is by default Unicode, this explicit instruction causes Linux to print mostly garbage
+            // Terminal.OutputEncoding = Encoding.Unicode;
```

Comment:
@pgrawehr do you think we need to uncomment this code when SWAN is running at Windows?

Reply:
I'll test again on windows, but I think it works correctly if you just don't touch the Console.OuputEncoding at all, it seems to work as it should with the default setting. I just added a comment because it _didn't_ work on Linux with this uncommented. Seems the style bot is not happy about that...

Reply:
OK!